lib/pickup_bot/commands/create_game.rb

```diff
@@ -0,0 +1,50 @@
+module Commands
+  class CreateGame
+    def self.run(telegram_bot, message)
+      new(telegram_bot, message).run
+    end
+
+    def initialize(telegram_bot, message)
+      @telegram_bot = telegram_bot
+      @message = message
+    end
+
+    def run
+      if game_exists?
+        telegram_bot.api.send_message(
+          chat_id: message.chat.id,
+          text: I18n.t('bot.commands.create_game.game_exists', username: username)
+        )
+      else
+        required_players = message.text.split(" ").second.to_i
+        game = Game.new(
+          chat_id: @message.chat.id,
+          name: game_name,
+          required_players: required_players
+          )
+        game.save
```

Comment:
instead of 
```ruby
game = Game.new(
   chat_id: @message.chat.id,
   name: game_name,
   required_players: required_players
)
game.save
```
you can do
```ruby
Game.create(
   chat_id: @message.chat.id,
   name: game_name,
   required_players: required_players
)
```